pkg/scheduler/api/resource_info.go

```diff
@@ -36,6 +36,13 @@ const (
 	minResource float64 = 0.1
 )
 
+const (
+	// Zero means resource dimension not defined will be treated as zero
+	Zero = "Zero"
+	// Infinity means resource dimension not defined will be treated as infinity
+	Infinity = "Infinity"
+)
+
```

Comment:
From my personal preference, I tend to define this as a new type, maybe something like
```
type Dimension string

const (
	DimensionZero Dimension= "Zero"
	DimensionInfinity Dimension = "Infinity"
)
```

Reply:
> From my personal preference, I tend to define this as a new type, maybe something like
> 
> ```
> type Dimension string
> 
> const (
> 	DimensionZero Dimension= "Zero"
> 	DimensionInfinity Dimension = "Infinity"
> )
> ```

That looks great. Let me optimize it.

Reply:
DimensionDefaultValue is ok for me